### PR TITLE
Updated mailbox.org

### PR DESCRIPTION
**Pull request by screendriver**

### CODE REVIEW

**Comment by screendriver** [comment]
It seems that there is an issue on Travis but I changed nothing special except one entry in a YAML file.

**Comment by stephengroat** [comment]
@screendriver just remove the Twitter handle, like travis says. No Twitter handle if tfa is enabled

**Comment by screendriver** [comment]
Oh sorry. I thought this was a link to the usual Twitter account of the provider.

I fixed that and Travis went green.

File: _data/email.yml [hunk 1]
```diff
@@ -61,8 +61,9 @@ websites:
       url: https://www.mailbox.org/
```
Comment (Carlgo11) [inline _data/email.yml]: Please change the url to `https://mailbox.org/`

**Comment by screendriver** [comment]
Fixed mailbox.org URL

**Pull request merged**